contracts/estate/EstateRegistry.sol

```diff
@@ -86,11 +103,23 @@ contract EstateRegistry is ERC721Token, Ownable, MetadataHolderBase, IEstateRegi
     return bytes4(0xf0b9e5ba);
   }
 
+  /**
+   * @notice Unlock any token sent to the contract without using safeTransferFrom
+   * @dev It uses the same logic as onERC721Received
+   * @param estateId The new owner of the locked token
+   * @param tokenId Locked token id
+   */
   function ammendReceived(uint256 estateId, uint256 tokenId) external {
     _pushTokenId(estateId, tokenId);
     emit AmmendReceived(estateId, tokenId);
   }
 
+  /**
+   * @notice Transfer a token owned by a estate to a new owner
+   * @param estateId Current owner of the token
+   * @param tokenId Token to be transfered
+   * @param destinatary New owner
+   */
   function transferToken(
```

Comment:
If we decide to use specific names we should consider change`transferToken` and `transferManyTokens` to
`transferLand` and `transferManyLand`

Reply:
The Estate can only own one type of Token for the moment (LAND), I think we are ok with transferToken as long as we don't have multiple token types.

Reply:
Cool, so why not call them `transferLand`?